### PR TITLE
Add FIPS validation wording.

### PR DESCRIPTION
**Pull request by richsalz**
We get asked often.  It will be easy to point to this page now.

### CODE REVIEW

File: docs/fips.html [hunk 1]
```diff
@@ -27,6 +27,19 @@
...
+            Neither validation will work with any release other than 1.0.2.
+            The OpenSSL project is no longer maintaining either the 1747
+            or the 2398 module. This includes adding platforms to those
+            validations.
+            We are starting work on a new validation, based on the upcoming
+            1.1.1 release.
+            That module will have a fixed set of validations.
+            The OpenSSL project is no longer involved in doing validations;
+            it is not our core competency, and any revenue we received
+            was not worth the time and resources it took.
```
Comment (iamamoose) [inline docs/fips.html]: Don't include "and any revenue we received was not worth the time and resources it took."

**Comment by t-j-h** [comment]
We are no longer involved in private label validations or platform updates to the current validation certificates (list the set of certificates).  

We are still very much active in FIPS validation work.

I wouldn't include any comments on competency at all. 

Comments about "not worth the time" etc don't belong in an FAQ entry.

**Comment by richsalz** [comment]
Additional commit pushed addressing @t-j-h 's feedback.

**Comment by richsalz** [comment]
ping @t-j-h

File: docs/faq-5-misc.txt [hunk 2]
```diff
@@ -33,6 +33,12 @@ that came with the version of OpenSSL you are using. The pod format
 documentation is included in each OpenSSL distribution under the docs
 directory.
 
+* I need a FIPS validated offering
+
+Please see
+@@@https://www.openssl.org/docs/fips.html@@@; the OpenSSL project no longer
+does FIPS validation work.
```
Comment (t-j-h) [inline docs/faq-5-misc.txt]: the OpenSSL project is no longer involved in private label validations nor adding platforms to the existing certificates

(to be consistent with the later statements and to be factually correct - the project is doing FIPS validation work as part of the next release)

File: docs/fips.html [hunk 3]
```diff
@@ -27,6 +27,18 @@
 	    (revision 2.0.16).
 	    </p>
 
+            <p>
+            Neither validation will work with any release other than 1.0.2.
+            The OpenSSL project is no longer maintaining either the 1747
+            or the 2398 module. This includes adding platforms to those
+            validations.
+            We are starting work on a new validation, based on the upcoming
+            1.1.1 release.
```
Comment (t-j-h) [inline docs/fips.html]: Technically "after the 1.1.1 release completes" is the better wording. 

This is not a 1.1.1 focused effort - it is for the next release after 1.1.1 which has not yet been labelled. I'm concerned that people will read that to mean we are committing to a 1.1.1 release FIPS solution which are **not** doing so.

File: docs/fips.html [hunk 4]
```diff
@@ -27,6 +27,18 @@
...
+            That module will have a fixed set of validations.
```
Comment (t-j-h) [inline docs/fips.html]: "a small set of validated operational environments (platforms)" 

is better than "fixed set of validations"

**Comment by t-j-h** [comment]
Three minor but significant tweaks and it will be good to go.

**Comment by richsalz** [comment]
Additional commit to address your latest feedback, @t-j-h

**Review by t-j-h — APPROVED** [review]
Thanks. Appreciate the patience on the multiple comments. Looks good!

**Comment by richsalz** [comment]
I appreciate the detailed feedback.  Squashed and merged, closing.

**Pull request closed without merging**